packages/web3-providers/src/zora/index.ts

```diff
@@ -0,0 +1,350 @@
+
+function createNFTAsset(asset: any): NonFungibleTokenAPI.Asset {
+    return {
+        is_verified: false,
+        is_auction: asset.auctions.length > 0,
+        image_url: asset.metadata.json.animation_url ?? asset.metadata.json.image_url ?? '',
+        asset_contract: {
+            name: asset.tokenContract.name,
+            description: '',
+            schemaName: '',
+        },
+        current_price: 0,
+        current_symbol: asset.symbol ?? 'ETH',
+        owner: asset.owner
+            ? {
+                  address: asset.owner,
+                  profile_img_url: '',
+                  user: { username: asset.owner },
+                  link: `https://zora.co/${asset.owner}`,
+              }
+            : null,
+        creator: asset.metadata.json.created_by
+            ? {
+                  address: asset.metadata.json.created_by,
+                  profile_img_url: '',
+                  user: { username: asset.metadata.json.created_by },
+                  link: `https://zora.co/${asset.metadata.json.created_by}`,
+              }
+            : null,
+        token_id: asset.token_id,
+        token_address: asset.address,
+        traits: asset.metadata.json.attributes,
+        safelist_request_status: '',
+        description: asset.metadata.json.description,
+        name: asset.name ?? asset.metadata.json.name,
+        collection_name: '',
+        animation_url: asset.metadata.json.animation_url,
+        end_time: asset.auctions.expiresAt ? new Date(asset.auctions.expiresAt) : null,
+        order_payment_tokens: [] as FungibleTokenDetailed[],
+        offer_payment_tokens: [] as FungibleTokenDetailed[],
+        slug: '',
+        top_ownerships: asset.owner
+            ? [
+                  {
+                      owner: {
+                          address: asset.owner,
+                          profile_img_url: '',
+                          user: { username: asset.owner },
+                          link: `https://zora.co/${asset.owner}`,
+                      },
+                  },
+              ]
+            : [],
+        response_: asset as any,
+        last_sale: null,
+    }
+}
+
+function createERC721TokenFromAsset(tokenAddress: string, tokenId: string, asset?: any): ERC721TokenDetailed {
+    return {
+        contractDetailed: {
+            type: EthereumTokenType.ERC721,
+            chainId: ChainId.Mainnet,
+            address: tokenAddress,
+            name: asset.tokenContract.name ?? '',
+            symbol: '',
+        },
+        info: {
+            name: asset.metadata.json.name ?? '',
+            description: asset.metadata.json.description ?? '',
+            mediaUrl: asset.metadata.json.animation_url ?? asset.metadata.json.image_url ?? '',
+            owner: asset.owner,
+        },
+        tokenId: tokenId,
+    }
+}
+
+export class ZoraApi implements NonFungibleTokenAPI.Provider {
+    private readonly _apiKey
```

Comment:
No using?

Reply:
Only OpenSea has this cause of the API requirements. But Zora API doesn't have a thing like that. It's  Not necessary.